src/pages/get-involved.js

```diff
@@ -0,0 +1,222 @@
+  }
+
+  ${mq(BreakPoint.sm)} {
+    .contact-items {
+      grid-template-columns: repeat(2, 1fr);
+    }
+
+  }
+`;
+
+const headerCss = css`
+  font-weight: 600;
+
+  .header {
+    height: 12.5rem;
+  }
+
+  p {
+    color: var(--white);
+  }
+
+  .highlight {
+    font-size: 1.25rem;
+  }
+
+  h2 {
+    background-color: var(--yellow);
+    color: var(--midnight);
+    display: inline-block;
+    padding: 0.125rem;
+  }
+
+  .container {
+    padding: 0.75rem;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    justify-content: center;
+    gap: 1rem;
+    height: 100%;
+  }
+
+  ${mq(BreakPoint.md)} {
+    .header {
+      padding: 0 2.5rem;
+      height: 16rem;
+    }
+    h2 {
+      font-size: 2rem;
+    }
+    .highlight {
+      font-size: 1.5rem;
+    }
+  }
+  ${mq(BreakPoint.lg)} {
+    .header {
+      padding: 0 10rem;
+      height: 18.75rem;
+    }
+  }
+`;
+
+const Header = () => (
+  <header css={headerCss}>
+    <HeroBackgroundImage className="header">
+      <div className="container">
+        <p>Home &gt; Get Involved</p>
+        <h2>Get Involved</h2>
+        <p className="highlight">&nbsp;</p>
+      </div>
+    </HeroBackgroundImage>
+  </header>)
+
+const Main = () => (
+  <main css={mainCss}>
+    <div className="heading">
+      <h1>Ways to get involved</h1>
+      <p>
+        Help us make the change. We have bold and ambitious plans and need help to execute them.
+      </p>
+    </div>
+    <div className="contact-items">
+      <div className="contact-item">
+        <h2>Newsletter</h2>
+        <p>Sign up to our newsletter to stay up to date on the progress of our plans, events and research.</p>
+        <Button label="Sign up" variant="light-grey" onClick={() => scroll.scrollToBottom() } />
+      </div>
+      <div className="contact-item">
+        <h2>Connecting organisations</h2>
+        <p>Connect ESMA to organisations that are tackling social mobility to include on our platform for others to collaborate with.</p>
+      </div>
+      <div className="contact-item">
+        <h2>Funding community</h2>
+        <p>Connect ESMA to organisations that are tackling social mobility to include on our platform for others to collaborate with.</p>
+      </div>
+      <div className="contact-item">
+        <h2>Resources</h2>
+        <p>Provide technology resources to continue to build out our platform services</p>
+      </div>
+    </div>
+    <div className="more-contact">
+      <p>If you’d like to know more about the above you can email:</p>
+      <div className="contact-panel">
+        <div className="more-contact-item">
+          <div className="image-placeholder" style={{backgroundColor: "pink", borderRadius: "9999px", width: "60px", height: "60px", float: "left", margin: "2.25em" }}></div>
+          <h2>Barry Matthews <a className="linked-in" href="https://www.linkedin.com/in/barry-matthews-0970704/"><img alt="linked-in" src={linkedInImage} /></a></h2>
```

Comment:
I don't think this should be a heading, but maybe it would be if there's more of a bio here.